distribution/ddf-common/src/main/resources/etc/org.ops4j.pax.logging.cfg

```diff
@@ -20,6 +20,8 @@
 # In order to switch to the xml-style configuration for Log4j2 in ${karaf.etc}/log4j2.config.xml, 
 # uncomment the following line and remove all the configuration beneath it. This change has not been 
 # made default because the xml-style configuration causes log entries to be missing from the itests.
+# Failover logging is not available using this configuration and can only be used by configuring
+# with the log4j2.config.xml file
```

Comment:
Advice on wording here? 
`Using the log4j2.config.xml file is a hardening step` ?